web/js/run.js

```diff
@@ -13,23 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-function addMessage(err, str) {
+function addMessage(type, str) {
     // Catch exceptions to protect against cross-domain access errors.
     try {
-        if (window.parent && window.parent.addToMessage) {
-            window.parent.addToMessage(str);
-
-            if (err) {
-                let message = window.parent.document.getElementById('message');
-                message.classList.add('error');
-            }
-
+        if (window.parent && window.parent.printMessage) {
+            window.parent.printMessage(type, str);
             return;
         }
     } catch (e) {}
-
-    console.log(str);
```

Comment:
Please leave this here.  This is here to make sure output messages are still accessible somehow when the program run outside of the CodeWorld environment.

Reply:
Sorry, I assumed it was mine forgotten log :smile: